### PR TITLE
Support/Add HUDI Token Icon

### PR DESCRIPTION
**Pull request by Skyflash**
### 📝 Description

Add Logo Icon of HUDI Token
Website: https://humandataincome.com

### ❓ Context

- **Impacted projects**: ``@ledgerhq/live-common``

Add logos for HUDI token. Support for the token was added to ledger live desktop already
Contract address: 0x83d8Ea5A4650B68Cd2b57846783d86DF940F7458

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** this PR is standalone and not depends by any other else
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .changeset/slow-maps-camp.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
```
Comment (aelmilligy-ledger) [inline .changeset/slow-maps-camp.md:2]: Hi @Skyflash 
Could you change "minor" to "patch"?

Comment (Skyflash) [reply .changeset/slow-maps-camp.md:2]: Ok. I have to make a new PR or i can modify this?

**Comment by gre** [comment]
👋  ESLint config has been unified on develop after we merged https://github.com/LedgerHQ/ledger-live/pull/3622 – this will improve the project consistency, remove eslint dependencies duplicates, and overall simplify the maintenance. This comes with a prettier library upgrade and unification of the prettierrc too, so you likely have conflicts with develop.


Hopefully, here is the procedure to easily fix this. **please proceed as follow**:

**Step 1: sync your branch with develop up to the config changes:**

2 cases

- if your branch were continuously rebased with develop and don't have any merge commit, you can do 
```
git rebase aa1e0ebc0a0498b4350754899b6a368fbd42e811
```
- otherwise prefer 
```
git merge aa1e0ebc0a0498b4350754899b6a368fbd42e811
```

**then, apply the config**

```
pnpm i
pnpm lint:fix --force
```

if there are remaining errors that lint couldn't fix, please fix them and `pnpm lint` again.

then commit in your branch.

```
git commit -a -m "chore: lint fixes"
```

**Step 2: finalize the rebase/merge with develop**

2 cases

- if your branch were continuously rebased with develop and don't have any merge commit, you can do
```
git pull --rebase origin develop
```
- otherwise prefer 
```
git merge origin/develop
```

you should now be up to date. 👍

---

*Thanks and sorry for the inconvenience,*
*gre*

File: .changeset/slow-maps-camp.md (hunk 1 above)
Comment (aelmilligy-ledger) [reply .changeset/slow-maps-camp.md:2]: Hey @Skyflash You can delete the old one

**Pull request closed without merging**